server/prisma/src/seedPPH_Contacts.ts

```diff
@@ -0,0 +1,116 @@
+import {PrismaClient} from '@prisma/client';
+const fs = require('fs');
+const yaml = require('js-yaml');
+
+/**
+ * Import contacts from YAML file to database
+ * @param {PrismaClient} prisma
+ */
+async function seedPPHContacts(prisma: PrismaClient) {
+  let totalContactsCount = 0;
+  let missingAccountsCount = 0;
+
+  try {
+    const contactsCount = await prisma.pphContacts.count();
+    if (contactsCount > 0) {
+      console.log('PPH_Contacts table already seeded.');
+      return;
+    }
+
+    const yamlData = fs.readFileSync('./prisma/yaml-seeder-data/PPH_contacts.yaml', 'utf8');
+    const data = yaml.load(yamlData);
+
+    for (const item of data) {
+      totalContactsCount++;
+      let canInsert = true;
+
+      if (item.account_id) {
+        const accountExists = await prisma.pphAccounts.findUnique({
+          where: {account_id: item.account_id},
+        });
+
+        if (!accountExists) {
+          missingAccountsCount++; // Increment missing accounts counter
+          canInsert = false;
+
+          if (process.env.ENV === 'local') {
+            console.log(`Skipping contact with ID ${item.contact_id} due to non-existing account with ID ${item.account_id}`);
+          }
```

Comment:
Any reason this is only logged when seeding locally? If it's matter of privacy, I'd be less concerned about logging ID values as opposed to PII, especially since presumably these are all logged in GCP somewhere which has restricted access. This does seem like something that would be nice to always have logged (same for the rest of the seedPPH files that have this kind of logic)

Reply:
This was a request from Jaeger's team, as before it was looping over every skipped instance and console logging for each instance (For example, if a instance of data couldn't make a relation to accounts, contacts and opportunities, there would be three lines output for one object). 
They wanted a summary in lieu of all of that output. 

_Otherwise_, the hold up here looks like the Events page on the client side of things. I'll look into contacting the team who is handling this.